resources/options_lookup.tsv

```diff
@@ -997,7 +997,12 @@ Insulation Unfinished Attic	"Roof R-49, Unvented"	ResidentialConstructionsUnfini
 		ResidentialAirflow	unfinished_attic_sla=0																																												
 Insulation Finished Roof	None																																														
 Insulation Finished Roof	Uninsulated	ResidentialConstructionsFinishedRoof	cavity_r=0	install_grade=1	cavity_depth=9.25	filled_cavity=false	framing_factor=0.07	drywall_thick_in=0.5	osb_thick_in=0.75	rigid_r=0.0																																					
+Insulation Finished Roof	R-7	ResidentialConstructionsFinishedRoof	cavity_r=7	install_grade=1	cavity_depth=9.25	filled_cavity=true	framing_factor=0.07	drywall_thick_in=0.5	osb_thick_in=0.75	rigid_r=0.0																																					
+Insulation Finished Roof	R-13	ResidentialConstructionsFinishedRoof	cavity_r=13	install_grade=1	cavity_depth=9.25	filled_cavity=true	framing_factor=0.07	drywall_thick_in=0.5	osb_thick_in=0.75	rigid_r=0.0																																					
+Insulation Finished Roof	R-19	ResidentialConstructionsFinishedRoof	cavity_r=19	install_grade=1	cavity_depth=9.25	filled_cavity=true	framing_factor=0.07	drywall_thick_in=0.5	osb_thick_in=0.75	rigid_r=0.0																																					
 Insulation Finished Roof	R-30	ResidentialConstructionsFinishedRoof	cavity_r=30	install_grade=1	cavity_depth=9.25	filled_cavity=true	framing_factor=0.07	drywall_thick_in=0.5	osb_thick_in=0.75	rigid_r=0.0																																					
+Insulation Finished Roof	R-38	ResidentialConstructionsFinishedRoof	cavity_r=38	install_grade=1	cavity_depth=9.25	filled_cavity=true	framing_factor=0.07	drywall_thick_in=0.5	osb_thick_in=0.75	rigid_r=0.0																																					
+Insulation Finished Roof	R-49	ResidentialConstructionsFinishedRoof	cavity_r=49	install_grade=1	cavity_depth=9.25	filled_cavity=true	framing_factor=0.07	drywall_thick_in=0.5	osb_thick_in=0.75	rigid_r=0.0																																					

```

Comment:
Can you update the `cavity_depth` values to preserve the R-value per inch? (based on the R-30 example: 30/9.25 = 3.2432432432 R/in)

Reply:
Updated to match BEopt cavity depth = 11.25 inches for R-38 and R-49